### PR TITLE
Updated Python Materials

### PR DESCRIPTION
**Pull request by sushant4191**
Added a few references to books written in hindi.

## What does this PR do?
Add resource(s)

### Why is this valuable (or not)?
This adds new information being in Hindi.

### How do we know it's really free?
It's a website and also has pdf from google drive.

### For book lists, is it a book? For course lists, is it a course? etc.
Both.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-hi.md [hunk 1]
```diff
@@ -8,3 +8,9 @@
 * [C++ Brief Notes \| Hindi](https://ehindistudy.com/2020/12/01/cpp-notes-in-hindi/) - Yugal Joshi
 * [C++ in Hindi \| Hindi](https://www.bccfalna.com/IOC-AllEBooks/CPPinHindi.pdf) - Kuldeep Chand (PDF)
 * [C++ Introduction Book \| Hindi](https://ncsmindia.com/wp-content/uploads/2012/04/c++-hindi.pdf) - NCMS India (PDF)
+
+
+### <a id="python"></a>Python
+
+* [Python in Hindi](https://drive.google.com/file/d/1_0rMxENpNJf6orWbiNZvd1zqKRJWAlBt/view) - Kuldeep Chand
```
Comment (shrn01) [inline books/free-programming-books-hi.md:15]: Drive links are not supported. Please provide official links to the resource

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: books/free-programming-books-hi.md [hunk 2]
```diff
@@ -8,3 +8,9 @@
 * [C++ Brief Notes \| Hindi](https://ehindistudy.com/2020/12/01/cpp-notes-in-hindi/) - Yugal Joshi
 * [C++ in Hindi \| Hindi](https://www.bccfalna.com/IOC-AllEBooks/CPPinHindi.pdf) - Kuldeep Chand (PDF)
 * [C++ Introduction Book \| Hindi](https://ncsmindia.com/wp-content/uploads/2012/04/c++-hindi.pdf) - NCMS India (PDF)
+
+
+### <a id="python"></a>Python
+
+* [Python in Hindi](https://www.khansir.co.in/python-notes-in-hindi-pdf) - Kuldeep Chand
```
Comment (shrn01) [inline books/free-programming-books-hi.md:15]: is this an authorized distribution?

Comment (sushant4191) [reply books/free-programming-books-hi.md:15]: Yes, the website is a trusted platform, contains other study materials along side. And has python book inlcuded in it.

File: books/free-programming-books-hi.md [hunk 3]
```diff
@@ -8,3 +8,9 @@
...
+* [Python Notes in Hindi](https://pythoninhindis.blogspot.com/2020/04/python-in-hindi-python-language-in-hindi.html)
```
Comment (eshellman) [inline books/free-programming-books-hi.md:16]: no download link there

**Pull request closed without merging**